src/Geno/C++/GUI/Widgets/TitleBar.cpp

```diff
@@ -41,6 +41,10 @@
 #include <numeric>
 #include <string>
 
+#if defined( __linux__ )
+#include "GUI/Platform/Liunx/X11/glfwX11.h"
+#endif // __linux__
```

Comment:
1. Put the ifdef in the header file so that we safely can include this header without errors
2. Linux is misspelled
3. Split the functions into "X11Drag.h" and "X11Resize.h" and just shove them directly under Platform/Linux/